### PR TITLE
AMLS-4170 | HVD Page split

### PR DESCRIPTION
**Pull request by owen-opencast**
Pull request includes the page split and related fixes - see [comments on ticket](https://jira.tools.tax.service.gov.uk/browse/AMLS-4170) for information around making all questions editable, and Date of Change redirects.

## How Has This Been Tested?

Manual testing, spec tests.
Manual testing included:

- Changing from yes -> no
- Changing from no -> yes and answering second question
- Changing from no -> yes and not answering second question (should see incomplete section), then re-entering section and completing.
- Date of Change redirects correctly on _Products_, _HowToSellGoods_, and _Excise Goods_ pages (see description on [ticket](https://jira.tools.tax.service.gov.uk/browse/AMLS-4170)).

Acceptance tests will need to be updated and run.

## Checklist

- [x] Breaking change (fix or feature that would cause existing functionality to change).
- [x] Build passing.
- [x] Unit tests have full coverage.
- [ ] Unit test approach and implementation has been reviewed with QA (testers).
- [x] Requires acceptance test run.
- [x] Appropriate labels added.
- [x] RELEASE NOTES ADDED.

### CODE REVIEW

File: app/utils/DateOfChangeHelper.scala [hunk 1]
```diff
@@ -16,15 +16,42 @@
...
 import play.api.i18n.Messages.Implicits._
+import play.api.mvc.Call
 
 trait DateOfChangeHelper {
 
+  case class DateOfChangeRedirect(call: Call)
+
+  object DateOfChangeRedirect {
+
+    val CHECK_YOUR_ANSWERS:String = "1"
```
Comment (matthewharris1) [inline app/utils/DateOfChangeHelper.scala]: Is there any chance we can change this to lowerCamelCase just to keep consistency throughout the service?

File: app/views/hvd/cash_payment_first_date.scala.html [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+                    f = f,
+                    p = "paymentDate",
+                    labelText = "hvd.cash.payment.date.title",
+                    hintText = "hvd.cash.payment.date.example",
+                    example = false
+                )
+            }
+        }
+
+        </div>
```
Comment (ajchilts) [inline app/views/hvd/cash_payment_first_date.scala.html]: Found a bug in the code in which the question is duplicated and also the error message is displayed three times when no date is entered. 

Suggest replacing lines 35 through 57 with the following;

       ```
@date(
          f = f,
          p = "paymentDate",
          labelText = "hvd.cash.payment.date.title",
          hintText = "hvd.cash.payment.date.example",
          legendHidden = true,
          example = false
       )
```

**Comment by owen-opencast** [comment]
Made the suggested changes. It passes unit tests, and I manually tested the validation on the date field.

**Comment by ajchilts** [comment]
Passed local testing 
Passed accessibility 
Passed automation 
Passed automation with encryption 

@Bongowhack - do we need to integration test this? The page split requires no migration. Will leave that up to you.

**Pull request merged**